airmozilla/main/static/main/js/eventstatus.js

```diff
@@ -0,0 +1,22 @@
+$(function () {
+    var TEN_SECONDS = 10000;
```

Comment:
This is a unfortunate variable name. It makes it "impossible" to change our minds to 12 seconds some day as we'd also need to change from `TEN_SECONDS` to `TWELVE_SECONDS` so what was the point of making it a variable?

Also, a variable that is only used once is usually an anti-pattern. The purpose of variables is re-use so you never type the value more than once. 

Shorter code with less stuff in it is easier to read. This file is only 22 lines long but imagine you're eyes are on the `setInterval(function() {... }, SOME_LENGTH)` line and you're debugging. Then you have to move your eyes (or use your editors Find function) to jump away from what you were doing to find out the value of `SOME_LENGTH`. It's much better to keep the code "compact". I.e. lines that are relevant to each other (and only each other) close. 

What I would do in this case is to just keep it simple with:
```javascript
var interval = setInterval(function () {
    refreshIfStatusChanges();
}, 1000000);
```